submissions/dafen173/js-dom/style.css

```diff
@@ -0,0 +1,163 @@
+
+
+.maincontainer {
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+
+
+.mainmenubtn {
+    background-color: #696969;
+    color: white;
+    cursor: pointer;
+    padding:20px;
+    
+}
+
+/*
+.mainmenubtn:hover {
+    background-color: green;
+    }
+*/   
+
+.dropdown {
+    /*position: relative;*/
+    /*display: inline-block;*/
+    float: left;
+    width: 20%;
+}
+
+.dropdown-child {
+    background: linear-gradient(to top, #FFFFFF, #696969);
+}
+
+
+
+
+.dropdown-child div {
```

Comment:
Avoid styling by HTML tag.
Try to use HTML tags for markup, class - styles, id - js.